lib/ansible/modules/packaging/os/flatpak_remote.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017 John Kwiatkoski
+# Copyright: (c) 2018 Alexander Bethke
+# Copyright: (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: flatpak_remote
+version_added: '2.6'
+requirements:
+- flatpak
+author:
+- John Kwiatkoski (@jaykayy)
+- Alexander Bethke (@oolongbrothers)
+short_description: Manage flatpak repository remotes
+description:
+- Manage flatpak repository remotes.
+options:
+  name:
+    description:
+    - When I(state) is set to C(present), I(name) is added as a remote for installing flatpaks.
+      When used with I(state=absent) the remote with thet name will be removed.
+    required: true
+  remote:
+    description:
+    - When I(state) is set to C(present), I(remote) url is added as a flatpak remote for the
+      specified installation C(method).
+      When used with I(state=absent), this is not required.
+    required: false
```

Comment:
If a parameter is not required, you don't need to add this.
It helps readability if only required parameters have the required option.